libs/sdk-react-native/ios/BreezSDKMapper.swift

```diff
@@ -442,10 +442,25 @@ class BreezSDKMapper {
             "feeRate": lspInformation.feeRate,
             "timeLockDelta": lspInformation.timeLockDelta,
             "minHtlcMsat": lspInformation.minHtlcMsat,
-            "channelFeePermyriad": lspInformation.channelFeePermyriad,
             "lspPubkey": lspInformation.lspPubkey,
-            "maxInactiveDuration": lspInformation.maxInactiveDuration,
-            "channelMinimumFeeMsat": lspInformation.channelMinimumFeeMsat
+            "openingFeeParamsList": dictionaryOf(openingFeeParamsMenu: lspInformation.openingFeeParamsList)
+        ]
+    }
+
+    static func dictionaryOf(openingFeeParams: OpeningFeeParams) -> [String: Any?] {
```

Comment:
There are no optional properties in OpeningFeeParams, so this can be `[String: Any]`

Reply:
Done as part of df9aa9e